js/__tests__/p384/security-v2.test.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable max-len */
+const {
+  JsonWebSignature,
+  JsonWebKey,
+} = require('@transmute/json-web-signature-2020');
+
+const jsigs = require('jsonld-signatures');
+const {AssertionProofPurpose} = jsigs.purposes;
+
+const {documentLoader} = require('../../__fixtures__/documentLoader');
+
+it('can sign and verify', async () => {
+  const key = await JsonWebKey.from(require('../../__fixtures__/keys/p384.json'));
+  const suite = new JsonWebSignature({
+    key,
+  });
+
+  const signedDocument = await jsigs.sign(
+    {
+      '@context': ['https://w3id.org/security/v3-unstable', {
+        schema: 'http://schema.org/',
```

Comment:
Ahh smart way to reduce loading the whole schema.org context. The method I've normally gone with is just caching one of the files downloadable here: https://schema.org/docs/developers.html